server/protocols/protocols-lmtp/src/main/java/org/apache/james/lmtpserver/hook/MailboxDeliverToRecipientHandler.java

```diff
@@ -105,6 +105,13 @@ public long size() {
                 .smtpReturnCode(SMTPRetCode.MAIL_OK)
                 .smtpDescription(DSNStatus.getStatus(DSNStatus.SUCCESS, DSNStatus.CONTENT_OTHER) + " Message received <" + recipient.asString() + ">")
                 .build();
+        } catch (OverQuotaException e) {
+            LOGGER.error("OverQuota error handling ", e);

```

Comment:
Imo this is eventually expected upon regular operation, INFO  is likely ok.

"{} is overquota" is likely a more infirmative message. Pass target username as a parameter too to maximize usefullness.